src/js/_store/modules/categories.js

```diff
@@ -83,5 +92,30 @@ export default {
     clearMessage({ commit }) {
       commit('clearMessage');
     },
+    confirmDeleteCategory({ commit }, payload) {
+      commit('confirmDeleteCategory', { payload });
+    },
+    deleteCategory({ commit, rootGetters, state }) {
+      return new Promise(resolve => {
+        commit('clearMessage');
+        const data = parseInt(state.deleteCategoryId.categoryId, 10);
```

Comment:
[imo]

上記の内容を修正してもらえるとこちらの記述も以下のようにシンプルにできるかなと思います

```suggestion
        const data = parseInt(state.deleteCategoryId, 10);
```

Reply:
ありがとうございます。
修正したのですが、
stateのdeleteCategoryをオブジェクト形式で書き換えたため、
この箇所を、
オブジェクトの取り出し方で記述しました。
結局長くなってしまったのですが、ここから
もし削れる箇所があるようでしたらご指摘いただきたいです、よろしくお願いいたします

Reply:
修正してもらった形が一番綺麗かなと思います！

Reply:
ありがとうございます！